src/modules/Shape/Shape.js

```diff
@@ -0,0 +1,51 @@
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import {
+  AutoControlledComponent as Component,
```

Comment:
Let's don't use `AutoControlledComponent` at now, let's back to `React.Component`.